components/ProfileForm/editor.js

```diff
@@ -79,10 +82,32 @@ export const ProfileEditor = ({ profile, setEditMode, mutateProfile }) => {
               type="title"
             />
             <FormHelperText>
-              Enter your current role and company, or anything that is relevant
+              Enter your current role or anything that is relevant
               as your title.
             </FormHelperText>
           </FormControl>
+          <FormControl>
+            <FormLabel htmlFor="company">Company</FormLabel>
+            <Input
+              {...register("company")}
+              id="company"
+              type="company"
+            />
+            <FormHelperText>
+              Enter your current company, if you like.
+            </FormHelperText>
+          </FormControl>
+          <FormControl>
+            <FormLabel htmlFor="seniority">Seniority</FormLabel>
+            <Input
+              {...register("seniority")}
+              id="seniority"
+              type="seniority"
```

Comment:
same as above comment, this would be a `number` (one could play around with the HTML5 validation rules to set a min, and a max)

optional: it could be a [Select](https://chakra-ui.com/docs/components/form/select) with a limited range e.g. 0, 1, 2, 3-5, 6-10, 11-14, +15 (totally arbitrary numbers...)

Reply:
Thanks! I will bring this up on the next call.

Reply:
Let's keep it simple for this PR